web/src/components/FOI/Home/Redlining.js

```diff
@@ -901,7 +903,11 @@ const Redlining = React.forwardRef(
                   requestid,
                   currentLayer.redactionlayerid,
                   annotObjs,
-                  (data) => {},
+                  (data) => {
+                    fetchAnnotationsInfo(requestid, currentLayer.name.toLowerCase(), (error) => {
+                      console.log("Error:", error);
+                    });
+                  },

```

Comment:
I'm re-syncing the annotationsInfo with the server whenever annotation updates have been made. Then the logic for sorting out the correct page flag is in the useEffect function below.